test/selenium/lib/view/successful_page/accessors.js

```diff
@@ -12,7 +12,14 @@ function SuccessfulPageAccessor(driver) {
 
 SuccessfulPageAccessor.prototype = {
    get successMessageLocator() {
-        return this.driver.findElement(SELECTORS.successMessage);
+        var driver = this.driver;
```

Comment:
Nit: This line is not necessary. Arrow functions can be used with this version of node.